webrtc.html

```diff
@@ -7716,18 +7716,6 @@ <h2>Methods</h2>
                   throw <code>InvalidModificationError</code>. This ensures that we always have
                   something to offer, regardless of <code><var>transceiver</var>.<a data-link-for="RTCRTPTransceiver">direction</a></code>.</p>
                 </li>
-                <li>
-                  <p>Let <var>codecCapabilities</var> be the union of
-                  <code>RTCRtpSender.getCapabilities(kind).codecs</code> and
-                  <code>RTCRtpReceiver.getCapabilities(kind).codecs</code>.</p>
-                </li>
-                <li>
-                  <p>For each <var>codec</var> in <var>codecs</var>,</p>
-                  <ol>
-                    <li>If <var>codec</var> is not in <var>codecCapabilities</var>, throw
-                    <code>InvalidModificationError</code>.</li>
```

Comment:
I don't think this section is redundant, however I think it needs to say the same thing as above that if the intersection "only contains RTX, RED or FEC codecs or is an empty set" we should throw, otherwise you might succeed because codec contains "FEC" even though you don't have a codec for both sending and receiving.

Reply:
If this section is updated to say that I do think that makes the first section redundant.

But we also need to check that there is no element in codecs that is not in either capability set, to make sure you cannot prefer a codec that you do not support.